fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
The original problem here was to get the text "Live" for `LiveKicker`, and "Breaking" for `BreakingNewsKicker`, and so on. You do have the types for this, and this is resolved for you from `Content`.

What the editors enter for a custom kicker is always going to be the type `String`. And, you have the appropriate types for that too.

So are we talking about converting the types to some text representation, or talking about what exists inside `FreeHtmlKicker`?